content/pt-br/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -0,0 +1,176 @@
+
+## CoreDNS
+
+CoreDNS é um servidor oficial de DNS de propósito geral que pode atuar como DNS do cluster,
+cumprindo com as [especificações DNS](https://github.com/kubernetes/dns/blob/master/docs/specification.md).
+
+### CoreDNS ConfigMap options
+
+CoreDNS é um servidor DNS que é modular e plugável, com plugins que adicionam novas funcionalidades.
+O servidor CoreDNS pode ser configurado por um [Corefile](https://coredns.io/2017/07/23/corefile-explained/),
+que é o arquivo de configuração do CoreDNS. Como administrador de cluster, você pode modificar o
+{{< glossary_tooltip text="ConfigMap" term_id="configmap" >}} para o arquivo Corefile do CoreDNS para
+mudar como o descobrimento de serviços DNS se comporta para esse cluster.
+
+Em Kubernetes, o CoreDNS é instalado com a seguinte configuração padrão do Corefile:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        errors
+        health {
+            lameduck 5s
+        }
+        ready
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+            pods insecure
+            fallthrough in-addr.arpa ip6.arpa
+            ttl 30
+        }
+        prometheus :9153
+        forward . /etc/resolv.conf
+        cache 30
+        loop
+        reload
+        loadbalance
+    }
+```
+
+A configuração do Corefile inclui os seguintes [plugins](https://coredns.io/plugins/) do CoreDNS:
+
+* [errors](https://coredns.io/plugins/errors/): Erros são registrados para stdout.
+* [health](https://coredns.io/plugins/health/): A saúde do CoreDNS é reportada para
+`http://localhost:8080/health`. Nesta sintaxe estendida, `lameduck` fará o processo
+insalubre, esperando por 5 segundos antes que o processo seja encerrado.
+* [ready](https://coredns.io/plugins/ready/): Um endpoint HTTP na porta 8181 retornará 200 OK, quando todos os plugins que são capazes de sinalizar prontidão tiverem feito isso.
+* [kubernetes](https://coredns.io/plugins/kubernetes/): O CoreDNS responderá a consultas DNS
+ baseado no IP dos Serviços e Pods. Você pode encontrar [mais detalhes em](https://coredns.io/plugins/kubernetes/).
+  sobre este plugin no site do CoreDNS.
+  * `ttl` permite que você defina um TTL personalizado para as respostas. O padrão é 5 segundos. O TTL mínimo permitido é de 0 segundos e o máximo é de 3600 segundos. Definir o TTL como 0 impedirá que os registros sejam armazenados em cache.
+  * A opção `pods insecure` é fornecida para retrocompatibilidade  com o `kube-dns`.
+  * Você pode usar a opção `pods verified`, que retorna um registro A somente se houver um pod no mesmo namespace com um IP correspondente.
+  * A opção `pods disabled` pode ser usada se você não usar registros de pod.
```

Comment:
nit: render Pod with a capital P in the following sentences:
* "se houver um Pod"
* "usar registros de Pod"